spec/requests/core_induction_programme/course_lesson_spec.rb

```diff
@@ -64,6 +65,56 @@
         expect(course_lesson[:course_module_id]).to eq(second_course_module[:id])
       end
     end
+
+    describe "GET /create-lesson" do
+      it "renders the new lesson page" do
+        get cip_create_lesson_path(cip)
+        expect(response).to render_template(:new)
+      end
+    end
+
+    describe "POST /create-lesson" do
+      it "creates a new lesson" do
+        expect {
+          post cip_create_lesson_path(cip), params: {
+            course_lesson: {
+              title: "new lesson title",
+              ect_summary: "new ect summary",
+              completion_time_in_minutes: "10",
+              course_module_id: create(:course_module).id,
+            },
+          }
+        }.to change(CourseLesson, :count).by(1)
+      end
+
+      it "redirects to the lesson" do
+        post cip_create_lesson_path(cip), params: {
+          course_lesson: {
+            title: "new lesson title",
+            ect_summary: "new ect summary",
+            completion_time_in_minutes: "10",
+            course_module_id: create(:course_module).id,
+          },
+        }
+
+        expect(response).to redirect_to(lesson_path(CourseLesson.last))
+      end
+
+      context "sad path" do
```

Comment:
Maybe let's call it "when given invalid data" or something like that ;)

Reply:
done